test/stdlib/ErrorBridged.swift

```diff
@@ -70,6 +70,8 @@ ErrorBridgingTests.test("NSCopying") {
   }
 }
 
+// Gated on the availability of NSKeyedArchiver.archivedData(withRootObject:).
+@available(macOS 10.11, iOS 9.0, tvOS 9.0, watchOS 2.0, *)
```

Comment:
Since those are the earliest public versions of tvOS and watchOS, it'd also be correct to leave them off, but it's up to you.

Reply:
We tend to explicitly place those on our APIs where relevant so I stuck them here for consistency; good to know, though! I'll leave them for now so I don't have to retrigger testing.